packages/utah-design-system/src/components/Drawer.tsx

```diff
@@ -103,6 +164,7 @@ export type DrawerProps = {
   className?: string;
   main?: boolean;
   overlayProps: DOMProps;
+  size?: 'small' | 'medium' | 'large' | 'extraLarge';
```

Comment:
What are your thoughts on extracting a type for size so we are consistent in the design system?

I think we use it one or two other places IIRC?